assets/helpers/stripeCheckout/stripeCheckoutActions.js

```diff
@@ -13,9 +13,11 @@ export type Action =
   | { type: 'SET_STRIPE_CHECKOUT_TOKEN', token: string }
   | { type: 'CLOSE_STRIPE_OVERLAY' }
   | { type: 'OPEN_STRIPE_OVERLAY' }
-  | { type: 'SET_STRIPE_AMOUNT', amount: number }
   ;
 
+type CombinedState = {
+  stripeCheckout: StripeCheckoutState,
+};
```

Comment:
This CombinedState seems like a boilerplate and it only has one field. Is it possible to avoid its definition? Additionally, in order to be consistent, I would put it in the reducer file.